cea/default.config

```diff
@@ -767,30 +767,30 @@ output-path = {general:scenario}/../..
 output-path.type = PathParameter
 output-path.help = Path to location where to store the new project
 
-[disconnected-cooling]
+[decentralized]
 AHUflag = false
 AHUflag.type = BooleanParameter
-AHUflag.help = this is a configuration flag, which influences the disconnected building calculations
+AHUflag.help = Flag for decentralized building calculations satisfying demand from only Air Handling
```

Comment:
Why call it "Flag for..."? The description still does not communicate the intent.

Did you mean: "Restrict to decentralized building calculations that satisfy demand with only Air Handling"? What does that even mean?

Reply:
The demand of a building is divided into three main parts:
1. Air Handling
2. Air Recirculation
3. Sensible Heating/Cooling

This flag satisfies the demand of only one part. The remaining needs to be satisfied by centralized

Reply:
so you mean, the flags are mutually exclusive?